.github/release-drafter.yml

```diff
@@ -1,4 +1,4 @@
-name-template: 'Sistent v$NEXT_PATCH_VERSION'
+name-template: '@layer5/sistent@v$NEXT_PATCH_VERSION'
```

Comment:
Does the name have to change or can we leave it prettified as “Sistent”? - The same thing that we do and all the other repos.